livecheck/utils/portage_test.py

```diff
@@ -0,0 +1,54 @@
+"""Configuration for Pytest."""
+import pytest
+from livecheck.utils.portage import sanitize_version
+
+
+def test_sanitize_version() -> None:
+    # Test cases for normalize_version function
+    # fmt: off
+    test_cases = [
+        ("v1.2.3", "1.2.3"),
```

Comment:
Instead of defining test cases here, use [`pytest.mark.parameterize`](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize) . This makes it so a single test function becomes n test functions without having to write them manually. That way we can see what test cases pass even after a failing one.